makefile.shade

```diff
@@ -98,7 +110,15 @@ var buildTarget = "compile"
                     blockLogger.StartBlock(blockName);
                 }
 
-                Log.Info(blockName);
+                if (IsLinux)
+                {
+                    Exec("copy", "-ar .build " + Path.Combine(repo, ".build"), "");
```

Comment:
@pranavkm if you don't want to fix that in this PR, please file a follow-up to update `shade\_copy.shade` in Korebuild. We don't use the Sake script but do use `File.Copy()`.